core/templates/components/question-directives/question-player/question-player.directive.html

```diff
@@ -182,6 +181,7 @@ <h3 class="oppia-question-player-header" translate="I18N_QUESTION_PLAYER_LEARN_M
   .oppia-question-player-progress .current-progress {
     background: #0D48A1;
     height: 16px;
+    width: <[$ctrl.currentProgress]>%;
```

Comment:
I don't think this will do. Does it actually work? You might need to use ng-style instead. /cc @srijanreddy98

Reply:
Please use ng-style. Ditto below.